Create 09-02-22_writeup.md (#138)
very-good-science/data-ethics-club: site/contents/write_ups/2022/09-02-22_writeup.md

# Data Ethics Club: [“You Social Scientists Love Mind Games”: Experimenting in the “divide” between data science and critical algorithm studies](https://journals.sagepub.com/doi/full/10.1177/2053951719833404)
<!--Please don't edit the info panel below-->

```{admonition} What's this? 
This is summary of Wednesday 9th February's Data Ethics Club discussion, where we discussed the paper [“You Social Scientists Love Mind Games”: Experimenting in the “divide” between data science and critical algorithm studies](https://journals.sagepub.com/doi/full/10.1177/2053951719833404) by David Moats and Nick Seaver.

The summary was written by Huw Day, who tried to synthesise everyone's contributions to this document and the discussion. "We" = "someone at Data Ethics Club". 
Nina Di Cara and Natalie Thurlby helped with the final edit.

```

## Introduction

> This week at Data Ethics Club we discussed the paper “You Social Scientists Love Mind Games”: Experimenting in the “divide” between data science and critical algorithm studies by David Moats and Nick Seaver. The paper examines the divide between and intersection between data scientists and social scientists using methodology more familiar to anthropologists than those with a STEM background:

Reviewer: ```suggestion
This week at Data Ethics Club we discussed the paper “You Social Scientists Love Mind Games”: Experimenting in the “divide” between data science and critical algorithm studies by David Moats and Nick Seaver. The paper examines the divide between and intersection between data scientists and social scientists using a methodology more familiar to anthropologists than those with a STEM background:
```

"we attempted to stage an encounter in which practicing data scientists were asked to analyze a corpus of critical social science literature about their work, using tools of textual analysis such as co-word and topic modelling. The idea was to provoke discussion both about the content of these texts and the possible limits of such analyses."

We discussed the authors' approach, the divides between social and data science and what we can do to bridge this gap. We see some reoccuring themes when data scientists try to bring their tools to bear on social science problems and vice versa: https://xkcd.com/1831/ 

> ## What do you think about the authors’ approach (framing of the experiment, the first vs the second attempt, tools suggested, their reflections, Nick’s email)?

Reviewer: Huw - depending how much you can be bothered/whether you have time(!) it would be nice to have a brief summary of what the experiments were that the paper described before it starts going through the questions - I'm sorry I missed this one so I don't know! But no worries if not - it was a while ago so you might not remember either :)

> We thought the experiment was interesting, getting data scientists to be reflective in a way that is not typical of their skillset. Some of us felt the experiment lacked specificity about what data scientists actually do (using quantitative methods to answer specific questions). Perhaps it could have been more successful by asking them to read the papers and discuss them, as opposed to getting them to use these tools to analyse them with quantative methods. It didn't seem like there was a way for the participants to succeed with their analysis. 

Reviewer: ```suggestion
We thought the experiment was interesting, getting data scientists to be reflective in a way that is not typical of their skillset, though some of us felt the experiment lacked specificity about what data scientists actually do (using quantitative methods to answer specific questions). 
Perhaps it could have been more successful by asking the data scientists to read the papers and discuss them, as opposed to getting them to analyse them with quantitative methods. It didn't seem like there was a way for the participants to succeed with their analysis. 
```

> On the other hand, maybe the utility of the paper was in the self reflective nature of the study? If the objective was to start a conversation rather than to get any particular analysis out of it, then there is still value in that. The training of the authors is ethonography and anthropology. This mode of reasoning is different to that of data scientists and the hallmarks of what the participants were meant to achieve is different. This means lower sample sizes a greater focus on individual feedback/insight, with the discussion of the experiment comeing from the point of view of statistical/scientific method.

Reviewer: ```suggestion
On the other hand, maybe the utility of the paper was in the self reflective nature of the study? If the objective was to start a conversation rather than to get any particular analysis out of it, then there is still value in that. The training of the authors is ethonography and anthropology. This mode of reasoning is different to that of data scientists and the hallmarks of what the participants were meant to achieve is different. This means lower sample sizes a greater focus on individual feedback/insight, with the discussion of the experiment coming from the point of view of statistical/scientific method.
```

The authors mention the power imbalances between data scientists and social scientists but didn't expand on it. We would have liked to have seen more on this. Those working in STEM, (for example MDs and data scientists), are expected to know the answer and to express certainty, when sometimes they just don't know. Perhaps the goal of the experiment was to tell stories, and so in that sense the experiment was successful.

## What other divides, binaries and camps should we pay attention to?

[There seem to be Two Cultures in Statistical Modeling](https://projecteuclid.org/journals/statistical-science/volume-16/issue-3/Statistical-Modeling--The-Two-Cultures-with-comments-and-a/10.1214/ss/1009213726.full):

"There are two cultures in the use of statistical modeling to reach conclusions from data. One assumes that the data are generated by a given stochastic data model. The other uses algorithmic models and treats the data mechanism as unknown."

> Data scientists are submerged in the technical details whilst social scientists have to consider the social context - how does the categories/labelling come about? Data scientists come from lots of different backgrounds. The paper focussed on data scientists in academia, whereas many are also in industry. Within industry there is also a divide between the marketing/sales side vs the people who actually make it happen.

Reviewer: ```suggestion
Data scientists are submerged in the technical details whilst social scientists have to consider the social context - how did the categories/labelling come about? Data scientists come from lots of different backgrounds. The paper focussed on data scientists in academia, whereas many are also in industry. Within industry there is also a divide between the marketing/sales side vs the people who are involved in the technical work.
```
